Sep 26 17:37:17 raspberrypi org.gtk.vfs.Daemon[899]: A connection to the bus can't be made
Sep 26 17:37:17 raspberrypi org.gtk.vfs.Daemon[899]: g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
Sep 26 17:37:35 raspberrypi iot[519]: **** IoT Raspberry Pi Sample has started ****
Sep 26 17:37:35 raspberrypi iot[519]: Config file not found. Going to Quickstart mode
Sep 26 17:37:36 raspberrypi iot[519]: Running in Quickstart mode
Sep 26 17:37:36 raspberrypi vncserver-x11[498]: ServerManager: Server started
Sep 26 17:37:39 raspberrypi vncserver-x11[498]: ConsoleDisplay: Cannot find a running X server on vt1
Sep 26 17:37:39 raspberrypi vncserver-x11[498]: AgentInitCheck: agent comms failure
Sep 26 17:37:40 raspberrypi vncserver-x11[498]: ConsoleDisplay: Found running X server (pid=718)
Sep 26 17:37:45 raspberrypi vncserver-x11[498]: AgentInitCheck: no response from agent
Sep 26 17:46:31 raspberrypi org.gtk.vfs.Daemon[921]: A connection to the bus can't be made
Sep 26 17:46:31 raspberrypi org.gtk.vfs.Daemon[921]: g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
Sep 26 17:46:52 raspberrypi iot[532]: **** IoT Raspberry Pi Sample has started ****
Sep 26 17:46:52 raspberrypi iot[532]: Config file not found. Going to Quickstart mode
Sep 26 17:46:52 raspberrypi iot[532]: Running in Quickstart mode
Sep 26 17:46:52 raspberrypi vncserver-x11[499]: ServerManager: Server started
Sep 26 17:46:54 raspberrypi vncserver-x11[499]: ConsoleDisplay: Cannot find a running X server on vt1
Sep 26 17:46:54 raspberrypi vncserver-x11[499]: AgentInitCheck: agent comms failure
Sep 26 17:46:56 raspberrypi vncserver-x11[499]: ConsoleDisplay: Found running X server (pid=711)
Sep 26 17:50:26 raspberrypi org.gtk.vfs.Daemon[897]: A connection to the bus can't be made
Sep 26 17:50:26 raspberrypi org.gtk.vfs.Daemon[897]: g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
Sep 26 17:50:45 raspberrypi iot[523]: **** IoT Raspberry Pi Sample has started ****
Sep 26 17:50:45 raspberrypi iot[523]: Config file not found. Going to Quickstart mode
Sep 26 17:50:45 raspberrypi iot[523]: Running in Quickstart mode
Sep 26 17:50:46 raspberrypi vncserver-x11[498]: ServerManager: Server started
Sep 26 17:50:48 raspberrypi vncserver-x11[498]: ConsoleDisplay: Cannot find a running X server on vt1
Sep 26 17:50:48 raspberrypi vncserver-x11[498]: AgentInitCheck: agent comms failure
Sep 26 17:50:50 raspberrypi vncserver-x11[498]: ConsoleDisplay: Found running X server (pid=739)
Sep 26 18:33:13 raspberrypi org.gtk.vfs.Daemon[873]: A connection to the bus can't be made
Sep 26 18:33:13 raspberrypi org.gtk.vfs.Daemon[873]: g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
Sep 26 18:33:31 raspberrypi iot[535]: **** IoT Raspberry Pi Sample has started ****
Sep 26 18:33:31 raspberrypi iot[535]: Config file not found. Going to Quickstart mode
Sep 26 18:33:31 raspberrypi iot[535]: Running in Quickstart mode
Sep 26 18:33:32 raspberrypi vncserver-x11[499]: ServerManager: Server started
Sep 26 18:33:33 raspberrypi vncserver-x11[499]: ConsoleDisplay: Cannot find a running X server on vt1
Sep 26 18:33:33 raspberrypi vncserver-x11[499]: AgentInitCheck: agent comms failure
Sep 26 18:33:35 raspberrypi vncserver-x11[499]: ConsoleDisplay: Found running X server (pid=713)
Sep 26 18:36:38 raspberrypi org.gtk.vfs.Daemon[897]: A connection to the bus can't be made
Sep 26 18:36:38 raspberrypi org.gtk.vfs.Daemon[897]: g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
Sep 26 18:37:00 raspberrypi iot[521]: **** IoT Raspberry Pi Sample has started ****
Sep 26 18:37:00 raspberrypi iot[521]: Config file not found. Going to Quickstart mode
Sep 26 18:37:00 raspberrypi iot[521]: Running in Quickstart mode
Sep 26 18:37:00 raspberrypi vncserver-x11[497]: ServerManager: Server started
Sep 26 18:37:02 raspberrypi vncserver-x11[497]: ConsoleDisplay: Cannot find a running X server on vt1
Sep 26 18:37:03 raspberrypi vncserver-x11[497]: AgentInitCheck: agent comms failure
Sep 26 18:37:04 raspberrypi vncserver-x11[497]: ConsoleDisplay: Found running X server (pid=701)
Sep 26 18:37:01 raspberrypi vncserver-x11[490]: ServerManager: Server started
Sep 26 18:37:02 raspberrypi vncserver-x11[490]: ConsoleDisplay: Cannot find a running X server on vt1
Sep 26 18:37:03 raspberrypi vncserver-x11[490]: AgentInitCheck: agent comms failure
Sep 26 18:37:05 raspberrypi vncserver-x11[490]: ConsoleDisplay: Found running X server (pid=716)
Sep 26 18:37:10 raspberrypi vncserver-x11[490]: AgentInitCheck: no response from agent
Sep 26 18:42:49 raspberrypi org.gtk.vfs.Daemon[933]: A connection to the bus can't be made
Sep 26 18:42:49 raspberrypi org.gtk.vfs.Daemon[933]: g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
Sep 26 18:43:11 raspberrypi iot[533]: **** IoT Raspberry Pi Sample has started ****
Sep 26 18:43:11 raspberrypi iot[533]: Config file not found. Going to Quickstart mode
Sep 26 18:43:11 raspberrypi iot[533]: Running in Quickstart mode
Sep 26 18:43:11 raspberrypi vncserver-x11[500]: ServerManager: Server started
Sep 26 18:43:13 raspberrypi vncserver-x11[500]: ConsoleDisplay: Cannot find a running X server on vt1
Sep 26 18:43:13 raspberrypi vncserver-x11[500]: AgentInitCheck: agent comms failure
Sep 26 18:43:15 raspberrypi vncserver-x11[500]: ConsoleDisplay: Found running X server (pid=709)
Sep 26 18:47:38 raspberrypi iot[529]: **** IoT Raspberry Pi Sample has started ****
Sep 26 18:47:38 raspberrypi iot[529]: Config file not found. Going to Quickstart mode
Sep 26 18:47:38 raspberrypi iot[529]: Running in Quickstart mode
Sep 26 18:47:39 raspberrypi vncserver-x11[497]: ServerManager: Server started
Sep 26 18:47:41 raspberrypi vncserver-x11[497]: ConsoleDisplay: Cannot find a running X server on vt1
Sep 26 18:47:41 raspberrypi vncserver-x11[497]: AgentInitCheck: agent comms failure
Sep 26 18:47:43 raspberrypi vncserver-x11[497]: ConsoleDisplay: Found running X server (pid=737)
Sep 26 18:47:48 raspberrypi vncserver-x11[497]: AgentInitCheck: no response from agent
Sep 26 19:18:42 raspberrypi org.gtk.vfs.Daemon[1049]: A connection to the bus can't be made
Sep 26 19:18:42 raspberrypi org.gtk.vfs.Daemon[1049]: g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
Sep 26 19:19:01 raspberrypi iot[531]: **** IoT Raspberry Pi Sample has started ****
Sep 26 19:19:01 raspberrypi iot[531]: Config file not found. Going to Quickstart mode
Sep 26 19:19:01 raspberrypi iot[531]: Running in Quickstart mode
Sep 26 19:19:02 raspberrypi vncserver-x11[502]: ServerManager: Server started
Sep 26 19:19:03 raspberrypi vncserver-x11[502]: ConsoleDisplay: Cannot find a running X server on vt1
Sep 26 19:19:04 raspberrypi vncserver-x11[502]: AgentInitCheck: agent comms failure
Sep 26 19:19:05 raspberrypi vncserver-x11[502]: ConsoleDisplay: Found running X server (pid=710)
Sep 26 19:19:10 raspberrypi vncserver-x11[502]: AgentInitCheck: no response from agent
Sep 26 19:21:04 raspberrypi org.gtk.vfs.Daemon[895]: A connection to the bus can't be made
Sep 26 19:21:04 raspberrypi org.gtk.vfs.Daemon[895]: g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
Sep 26 19:21:27 raspberrypi iot[527]: **** IoT Raspberry Pi Sample has started ****
Sep 26 19:21:27 raspberrypi iot[527]: Config file not found. Going to Quickstart mode
Sep 26 19:21:27 raspberrypi iot[527]: Running in Quickstart mode
Sep 26 19:21:27 raspberrypi vncserver-x11[500]: ServerManager: Server started
Sep 26 19:21:30 raspberrypi vncserver-x11[500]: ConsoleDisplay: Cannot find a running X server on vt1
Sep 26 19:21:30 raspberrypi vncserver-x11[500]: AgentInitCheck: agent comms failure
Sep 26 19:21:31 raspberrypi vncserver-x11[500]: ConsoleDisplay: Found running X server (pid=728)
Sep 26 19:21:28 raspberrypi vncserver-x11[489]: ServerManager: Server started
Sep 26 19:21:30 raspberrypi vncserver-x11[489]: ConsoleDisplay: Cannot find a running X server on vt1
Sep 26 19:21:30 raspberrypi vncserver-x11[489]: AgentInitCheck: agent comms failure
Sep 26 19:21:32 raspberrypi vncserver-x11[489]: ConsoleDisplay: Found running X server (pid=711)
Sep 26 19:21:37 raspberrypi vncserver-x11[489]: AgentInitCheck: no response from agent
Sep 26 19:21:28 raspberrypi vncserver-x11[497]: ServerManager: Server started
Sep 26 19:21:29 raspberrypi vncserver-x11[497]: ConsoleDisplay: Cannot find a running X server on vt1
Sep 26 19:21:30 raspberrypi vncserver-x11[497]: AgentInitCheck: agent comms failure
Sep 26 19:21:32 raspberrypi vncserver-x11[497]: ConsoleDisplay: Found running X server (pid=721)
Sep 26 19:21:37 raspberrypi vncserver-x11[497]: AgentInitCheck: no response from agent
Sep 26 19:39:37 raspberrypi org.gtk.vfs.Daemon[935]: A connection to the bus can't be made
Sep 26 19:39:37 raspberrypi org.gtk.vfs.Daemon[935]: g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
Sep 26 19:39:57 raspberrypi iot[533]: **** IoT Raspberry Pi Sample has started ****
Sep 26 19:39:57 raspberrypi iot[533]: Config file not found. Going to Quickstart mode
Sep 26 19:39:57 raspberrypi iot[533]: Running in Quickstart mode
Sep 26 19:39:57 raspberrypi vncserver-x11[496]: ServerManager: Server started
Sep 26 19:40:00 raspberrypi vncserver-x11[496]: ConsoleDisplay: Cannot find a running X server on vt1
Sep 26 19:40:00 raspberrypi vncserver-x11[496]: AgentInitCheck: agent comms failure
Sep 26 19:40:01 raspberrypi vncserver-x11[496]: ConsoleDisplay: Found running X server (pid=725)
Sep 26 20:01:17 raspberrypi iot[515]: **** IoT Raspberry Pi Sample has started ****
Sep 26 20:01:17 raspberrypi iot[515]: Config file not found. Going to Quickstart mode
Sep 26 20:01:17 raspberrypi iot[515]: Running in Quickstart mode
Sep 26 20:01:18 raspberrypi vncserver-x11[499]: ServerManager: Server started
Sep 26 20:01:20 raspberrypi vncserver-x11[499]: ConsoleDisplay: Cannot find a running X server on vt1
Sep 26 20:01:20 raspberrypi vncserver-x11[499]: AgentInitCheck: agent comms failure
Sep 26 20:01:22 raspberrypi vncserver-x11[499]: ConsoleDisplay: Found running X server (pid=714)
Sep 26 20:01:27 raspberrypi vncserver-x11[499]: AgentInitCheck: no response from agent
Sep 26 20:01:17 raspberrypi vncserver-x11[491]: ServerManager: Server started
Sep 26 20:01:19 raspberrypi vncserver-x11[491]: ConsoleDisplay: Cannot find a running X server on vt1
Sep 26 20:01:20 raspberrypi vncserver-x11[491]: AgentInitCheck: agent comms failure
Sep 26 20:01:22 raspberrypi vncserver-x11[491]: ConsoleDisplay: Found running X server (pid=713)
Sep 26 20:01:27 raspberrypi vncserver-x11[491]: AgentInitCheck: no response from agent
Sep 26 20:01:18 raspberrypi vncserver-x11[491]: ServerManager: Server started
Sep 26 20:01:20 raspberrypi vncserver-x11[491]: ConsoleDisplay: Cannot find a running X server on vt1
Sep 26 20:01:20 raspberrypi vncserver-x11[491]: AgentInitCheck: agent comms failure
Sep 26 20:01:22 raspberrypi vncserver-x11[491]: ConsoleDisplay: Found running X server (pid=718)
Sep 26 20:01:27 raspberrypi vncserver-x11[491]: AgentInitCheck: no response from agent
Sep 27 12:17:14 raspberrypi vncserver-x11[494]: ServerManager: Server started
Sep 27 12:17:16 raspberrypi vncserver-x11[494]: ConsoleDisplay: Cannot find a running X server on vt1
Sep 27 12:17:17 raspberrypi vncserver-x11[494]: AgentInitCheck: agent comms failure
Sep 27 12:17:18 raspberrypi vncserver-x11[494]: ConsoleDisplay: Found running X server (pid=713)
Sep 27 12:17:24 raspberrypi vncserver-x11[494]: AgentInitCheck: no response from agent
Sep 27 14:17:16 raspberrypi vncserver-x11[491]: ServerManager: Server started
Sep 27 14:17:17 raspberrypi vncserver-x11[491]: ConsoleDisplay: Cannot find a running X server on vt1
Sep 27 14:17:18 raspberrypi vncserver-x11[491]: AgentInitCheck: agent comms failure
Sep 27 14:17:20 raspberrypi vncserver-x11[491]: ConsoleDisplay: Found running X server (pid=713)
Sep 27 14:17:25 raspberrypi vncserver-x11[491]: AgentInitCheck: no response from agent
Sep 27 14:17:16 raspberrypi vncserver-x11[492]: ServerManager: Server started
Sep 27 14:17:17 raspberrypi vncserver-x11[492]: ConsoleDisplay: Cannot find a running X server on vt1
Sep 27 14:17:18 raspberrypi vncserver-x11[492]: AgentInitCheck: agent comms failure
Sep 27 14:17:20 raspberrypi vncserver-x11[492]: ConsoleDisplay: Found running X server (pid=711)
Sep 27 14:17:25 raspberrypi vncserver-x11[492]: AgentInitCheck: no response from agent
Sep 27 19:17:14 raspberrypi vncserver-x11[491]: ServerManager: Server started
Sep 27 19:17:16 raspberrypi vncserver-x11[491]: ConsoleDisplay: Cannot find a running X server on vt1
Sep 27 19:17:17 raspberrypi vncserver-x11[491]: AgentInitCheck: agent comms failure
Sep 27 19:17:18 raspberrypi vncserver-x11[491]: ConsoleDisplay: Found running X server (pid=713)
Sep 27 19:17:24 raspberrypi vncserver-x11[491]: AgentInitCheck: no response from agent
Sep 27 19:17:15 raspberrypi vncserver-x11[490]: ServerManager: Server started
Sep 27 19:17:16 raspberrypi vncserver-x11[490]: ConsoleDisplay: Cannot find a running X server on vt1
Sep 27 19:17:17 raspberrypi vncserver-x11[490]: AgentInitCheck: agent comms failure
Sep 27 19:17:19 raspberrypi vncserver-x11[490]: ConsoleDisplay: Found running X server (pid=722)
Sep 27 19:17:24 raspberrypi vncserver-x11[490]: AgentInitCheck: no response from agent
Sep 28 17:52:39 raspberrypi vncserver-x11[490]: Connections: connected: 10.255.255.249::61992
Sep 28 17:52:44 raspberrypi vncserver-x11[490]: Connections: authenticated: 10.255.255.249::61992, as pi (f permissions)
Sep 28 17:52:45 raspberrypi vncserver-x11[490]: SPrintConnMgr: Failed to add printer: server-error-service-unavailable
Sep 28 18:21:48 raspberrypi org.gtk.vfs.Daemon[936]: A connection to the bus can't be made
Sep 28 18:21:48 raspberrypi org.gtk.vfs.Daemon[936]: g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
Sep 28 18:21:48 raspberrypi vncserver-x11[490]: SPrintConnMgr: Failed to set allowed users: server-error-service-unavailable
Sep 28 18:22:06 raspberrypi iot[514]: **** IoT Raspberry Pi Sample has started ****
Sep 28 18:22:06 raspberrypi iot[514]: Config file not found. Going to Quickstart mode
Sep 28 18:22:06 raspberrypi iot[514]: Running in Quickstart mode
Sep 28 18:22:07 raspberrypi vncserver-x11[496]: ServerManager: Server started
Sep 28 18:22:08 raspberrypi vncserver-x11[496]: ConsoleDisplay: Cannot find a running X server on vt1
Sep 28 18:22:09 raspberrypi vncserver-x11[496]: AgentInitCheck: agent comms failure
Sep 28 18:22:11 raspberrypi vncserver-x11[496]: ConsoleDisplay: Found running X server (pid=716)
Sep 28 18:22:16 raspberrypi vncserver-x11[496]: Connections: connected: 10.255.255.249::62050
Sep 28 18:22:17 raspberrypi vncserver-x11[496]: Connections: authenticated: 10.255.255.249::62050, as pi (f permissions)
Sep 28 18:22:17 raspberrypi vncserver-x11[496]: SPrintConnMgr: Failed to add printer: server-error-service-unavailable
Sep 28 18:22:19 raspberrypi vncserver-x11[496]: SPrintConnMgr: Failed to set allowed users: server-error-service-unavailable
Sep 28 18:24:25 raspberrypi org.gtk.vfs.Daemon[880]: A connection to the bus can't be made
Sep 28 18:24:25 raspberrypi org.gtk.vfs.Daemon[880]: g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
Sep 28 18:24:25 raspberrypi vncserver-x11[496]: SPrintConnMgr: Failed to set allowed users: server-error-service-unavailable
Sep 28 18:24:43 raspberrypi iot[522]: **** IoT Raspberry Pi Sample has started ****
Sep 28 18:24:43 raspberrypi iot[522]: Config file not found. Going to Quickstart mode
Sep 28 18:24:43 raspberrypi iot[522]: Running in Quickstart mode
Sep 28 18:24:43 raspberrypi vncserver-x11[496]: ServerManager: Server started
Sep 28 18:24:46 raspberrypi vncserver-x11[496]: ConsoleDisplay: Cannot find a running X server on vt1
Sep 28 18:24:46 raspberrypi vncserver-x11[496]: AgentInitCheck: agent comms failure
Sep 28 18:24:47 raspberrypi vncserver-x11[496]: ConsoleDisplay: Found running X server (pid=717)
Sep 28 18:24:52 raspberrypi vncserver-x11[496]: AgentInitCheck: no response from agent
Sep 28 18:25:54 raspberrypi vncserver-x11[496]: Connections: connected: 10.255.255.249::62060
Sep 28 18:25:55 raspberrypi vncserver-x11[496]: Connections: authenticated: 10.255.255.249::62060, as pi (f permissions)
Sep 28 18:25:56 raspberrypi vncserver-x11[496]: SPrintConnMgr: Failed to add printer: server-error-service-unavailable
Sep 28 18:27:38 raspberrypi org.gtk.vfs.Daemon[920]: A connection to the bus can't be made
Sep 28 18:27:39 raspberrypi org.gtk.vfs.Daemon[920]: g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
Sep 28 18:27:56 raspberrypi iot[517]: **** IoT Raspberry Pi Sample has started ****
Sep 28 18:27:56 raspberrypi iot[517]: Config file not found. Going to Quickstart mode
Sep 28 18:27:56 raspberrypi iot[517]: Running in Quickstart mode
Sep 28 18:27:57 raspberrypi vncserver-x11[497]: ServerManager: Server started
Sep 28 18:27:59 raspberrypi vncserver-x11[497]: ConsoleDisplay: Cannot find a running X server on vt1
Sep 28 18:28:00 raspberrypi vncserver-x11[497]: AgentInitCheck: agent comms failure
Sep 28 18:28:01 raspberrypi vncserver-x11[497]: ConsoleDisplay: Found running X server (pid=750)
Sep 28 18:28:01 raspberrypi vncserver-x11[497]: Connections: connected: 10.255.255.249::62067
Sep 28 18:28:03 raspberrypi vncserver-x11[497]: Connections: authenticated: 10.255.255.249::62067, as pi (f permissions)
Sep 28 18:28:04 raspberrypi vncserver-x11[497]: SPrintConnMgr: Failed to add printer: server-error-service-unavailable
Sep 28 18:28:06 raspberrypi vncserver-x11[497]: AgentInitCheck: no response from agent
Sep 28 18:28:26 raspberrypi vncserver-x11[497]: SPrintConnMgr: Failed to set allowed users: server-error-service-unavailable
Sep 28 18:28:27 raspberrypi vncserver-x11[497]: SPrintConnMgr: Failed to set allowed users: server-error-service-unavailable
Sep 28 18:28:36 raspberrypi org.gtk.Private.AfcVolumeMonitor[1065]: Volume monitor alive
Sep 28 18:27:57 raspberrypi vncserver-x11[502]: ServerManager: Server started
Sep 28 18:27:59 raspberrypi vncserver-x11[502]: ConsoleDisplay: Cannot find a running X server on vt1
Sep 28 18:28:00 raspberrypi vncserver-x11[502]: AgentInitCheck: agent comms failure
Sep 28 18:28:01 raspberrypi vncserver-x11[502]: ConsoleDisplay: Found running X server (pid=702)
Oct  1 14:29:40 raspberrypi org.gtk.Private.AfcVolumeMonitor[874]: Volume monitor alive
Oct  1 14:47:40 raspberrypi iot[536]: **** IoT Raspberry Pi Sample has started ****
Oct  1 14:47:40 raspberrypi iot[536]: Config file not found. Going to Quickstart mode
Oct  1 14:47:41 raspberrypi iot[536]: Running in Quickstart mode
Oct  1 14:47:41 raspberrypi vncserver-x11[507]: ServerManager: Server started
Oct  1 14:47:43 raspberrypi vncserver-x11[507]: ConsoleDisplay: Cannot find a running X server on vt1
Oct  1 14:47:43 raspberrypi vncserver-x11[507]: AgentInitCheck: agent comms failure
Oct  1 14:47:45 raspberrypi vncserver-x11[507]: ConsoleDisplay: Found running X server (pid=729)
Oct  1 14:47:50 raspberrypi vncserver-x11[507]: AgentInitCheck: no response from agent
Oct  1 16:00:17 raspberrypi org.gtk.Private.AfcVolumeMonitor[926]: Volume monitor alive
Oct  1 16:40:52 raspberrypi org.gtk.vfs.Daemon[926]: A connection to the bus can't be made
Oct  1 16:40:52 raspberrypi org.gtk.vfs.Daemon[926]: g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
Oct  1 16:41:12 raspberrypi vncserver-x11[485]: ServerManager: Server started
Oct  1 16:41:14 raspberrypi vncserver-x11[485]: ConsoleDisplay: Cannot find a running X server on vt1
Oct  1 16:41:14 raspberrypi vncserver-x11[485]: AgentInitCheck: agent comms failure
Oct  1 16:41:17 raspberrypi iot[599]: **** IoT Raspberry Pi Sample has started ****
Oct  1 16:41:17 raspberrypi iot[599]: Config file not found. Going to Quickstart mode
Oct  1 16:41:17 raspberrypi iot[599]: Running in Quickstart mode
Oct  1 16:41:18 raspberrypi vncserver-x11[485]: ConsoleDisplay: Found running X server (pid=628)
Oct  1 16:41:24 raspberrypi vncserver-x11[485]: AgentInitCheck: no response from agent
Oct  1 17:42:35 raspberrypi org.gtk.Private.AfcVolumeMonitor[971]: Volume monitor alive
Oct  1 18:20:19 raspberrypi org.gtk.vfs.Daemon[971]: A connection to the bus can't be made
Oct  1 18:20:39 raspberrypi iot[523]: **** IoT Raspberry Pi Sample has started ****
Oct  1 18:20:39 raspberrypi iot[523]: Config file not found. Going to Quickstart mode
Oct  1 18:20:39 raspberrypi iot[523]: Running in Quickstart mode
Oct  1 18:20:39 raspberrypi vncserver-x11[493]: ServerManager: Server started
Oct  1 18:20:41 raspberrypi vncserver-x11[493]: ConsoleDisplay: Cannot find a running X server on vt1
Oct  1 18:20:41 raspberrypi vncserver-x11[493]: AgentInitCheck: agent comms failure
Oct  1 18:20:43 raspberrypi vncserver-x11[493]: ConsoleDisplay: Found running X server (pid=719)
Oct  1 18:21:03 raspberrypi org.gtk.Private.AfcVolumeMonitor[895]: Volume monitor alive
Oct  1 18:42:02 raspberrypi org.gtk.vfs.Daemon[895]: A connection to the bus can't be made
Oct  1 18:42:02 raspberrypi org.gtk.vfs.Daemon[895]: g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
Oct  1 18:42:19 raspberrypi iot[541]: **** IoT Raspberry Pi Sample has started ****
Oct  1 18:42:19 raspberrypi iot[541]: Config file not found. Going to Quickstart mode
Oct  1 18:42:19 raspberrypi iot[541]: Running in Quickstart mode
Oct  1 18:42:20 raspberrypi vncserver-x11[513]: ServerManager: Server started
Oct  1 18:42:22 raspberrypi vncserver-x11[513]: ConsoleDisplay: Cannot find a running X server on vt1
Oct  1 18:42:22 raspberrypi vncserver-x11[513]: AgentInitCheck: agent comms failure
Oct  1 18:42:24 raspberrypi vncserver-x11[513]: ConsoleDisplay: Found running X server (pid=712)
Oct  1 18:42:46 raspberrypi org.gtk.Private.AfcVolumeMonitor[915]: Volume monitor alive
Oct  1 18:44:38 raspberrypi org.gtk.vfs.Daemon[915]: A connection to the bus can't be made
Oct  1 18:44:38 raspberrypi org.gtk.vfs.Daemon[915]: g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
Oct  1 18:45:06 raspberrypi iot[526]: **** IoT Raspberry Pi Sample has started ****
Oct  1 18:45:06 raspberrypi iot[526]: Config file not found. Going to Quickstart mode
Oct  1 18:45:06 raspberrypi iot[526]: Running in Quickstart mode
Oct  1 18:45:06 raspberrypi vncserver-x11[499]: ServerManager: Server started
Oct  1 18:45:08 raspberrypi vncserver-x11[499]: ConsoleDisplay: Cannot find a running X server on vt1
Oct  1 18:45:09 raspberrypi vncserver-x11[499]: AgentInitCheck: agent comms failure
Oct  1 18:45:10 raspberrypi vncserver-x11[499]: ConsoleDisplay: Found running X server (pid=734)
Oct  1 18:45:49 raspberrypi org.gtk.Private.AfcVolumeMonitor[870]: Volume monitor alive
Oct  1 18:45:06 raspberrypi vncserver-x11[502]: ServerManager: Server started
Oct  1 18:45:08 raspberrypi vncserver-x11[502]: ConsoleDisplay: Cannot find a running X server on vt1
Oct  1 18:45:08 raspberrypi vncserver-x11[502]: AgentInitCheck: agent comms failure
Oct  1 18:45:10 raspberrypi vncserver-x11[502]: ConsoleDisplay: Found running X server (pid=711)
Oct  1 18:45:29 raspberrypi org.gtk.Private.AfcVolumeMonitor[893]: Volume monitor alive
Oct  1 19:01:03 raspberrypi vncserver-x11[502]: Connections: connected: 10.255.255.249::64836
Oct  1 19:01:07 raspberrypi vncserver-x11[502]: Connections: authenticated: 10.255.255.249::64836, as pi (f permissions)
Oct  1 19:01:08 raspberrypi vncserver-x11[502]: SPrintConnMgr: Failed to add printer: server-error-service-unavailable
Oct  1 19:03:58 raspberrypi org.gtk.vfs.Daemon[893]: A connection to the bus can't be made
Oct  1 19:03:58 raspberrypi org.gtk.vfs.Daemon[893]: g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
Oct  1 19:03:58 raspberrypi vncserver-x11[502]: SPrintConnMgr: Failed to set allowed users: server-error-service-unavailable
Oct  1 19:04:16 raspberrypi iot[529]: **** IoT Raspberry Pi Sample has started ****
Oct  1 19:04:16 raspberrypi iot[529]: Config file not found. Going to Quickstart mode
Oct  1 19:04:16 raspberrypi iot[529]: Running in Quickstart mode
Oct  1 19:04:16 raspberrypi vncserver-x11[495]: ServerManager: Server started
Oct  1 19:04:18 raspberrypi vncserver-x11[495]: ConsoleDisplay: Cannot find a running X server on vt1
Oct  1 19:04:18 raspberrypi vncserver-x11[495]: AgentInitCheck: agent comms failure
Oct  1 19:04:20 raspberrypi vncserver-x11[495]: ConsoleDisplay: Found running X server (pid=713)
Oct  1 19:04:48 raspberrypi org.gtk.Private.AfcVolumeMonitor[860]: Volume monitor alive
Oct  1 19:04:16 raspberrypi vncserver-x11[497]: ServerManager: Server started
Oct  1 19:04:18 raspberrypi vncserver-x11[497]: ConsoleDisplay: Cannot find a running X server on vt1
Oct  1 19:04:18 raspberrypi vncserver-x11[497]: AgentInitCheck: agent comms failure
Oct  1 19:04:20 raspberrypi vncserver-x11[497]: ConsoleDisplay: Found running X server (pid=708)
Oct  1 19:04:41 raspberrypi org.gtk.Private.AfcVolumeMonitor[1051]: Volume monitor alive
Oct  1 19:04:16 raspberrypi vncserver-x11[497]: ServerManager: Server started
Oct  1 19:04:18 raspberrypi vncserver-x11[497]: ConsoleDisplay: Cannot find a running X server on vt1
Oct  1 19:04:18 raspberrypi vncserver-x11[497]: AgentInitCheck: agent comms failure
Oct  1 19:04:20 raspberrypi vncserver-x11[497]: ConsoleDisplay: Found running X server (pid=711)
Oct  1 19:04:25 raspberrypi vncserver-x11[497]: AgentInitCheck: no response from agent
Oct  1 19:04:42 raspberrypi org.gtk.Private.AfcVolumeMonitor[1166]: Volume monitor alive
Oct  1 19:04:16 raspberrypi vncserver-x11[503]: ServerManager: Server started
Oct  1 19:04:18 raspberrypi vncserver-x11[503]: ConsoleDisplay: Cannot find a running X server on vt1
Oct  1 19:04:18 raspberrypi vncserver-x11[503]: AgentInitCheck: agent comms failure
Oct  1 19:04:20 raspberrypi vncserver-x11[503]: ConsoleDisplay: Found running X server (pid=702)
Oct  1 19:04:39 raspberrypi org.gtk.Private.AfcVolumeMonitor[894]: Volume monitor alive
Oct  1 19:15:37 raspberrypi vncserver-x11[503]: Connections: connected: 10.255.255.249::49624
Oct  1 19:15:41 raspberrypi vncserver-x11[503]: Connections: authenticated: 10.255.255.249::49624, as pi (f permissions)
Oct  1 19:15:42 raspberrypi vncserver-x11[503]: SPrintConnMgr: Failed to add printer: server-error-service-unavailable
Oct  1 19:18:24 raspberrypi org.gtk.vfs.Daemon[894]: A connection to the bus can't be made
Oct  1 19:18:25 raspberrypi vncserver-x11[503]: SPrintConnMgr: Failed to set allowed users: server-error-service-unavailable
Oct  1 19:18:42 raspberrypi iot[518]: **** IoT Raspberry Pi Sample has started ****
Oct  1 19:18:42 raspberrypi iot[518]: Config file not found. Going to Quickstart mode
Oct  1 19:18:42 raspberrypi iot[518]: Running in Quickstart mode
Oct  1 19:18:42 raspberrypi vncserver-x11[492]: ServerManager: Server started
Oct  1 19:18:45 raspberrypi vncserver-x11[492]: ConsoleDisplay: Cannot find a running X server on vt1
Oct  1 19:18:45 raspberrypi vncserver-x11[492]: AgentInitCheck: agent comms failure
Oct  1 19:18:46 raspberrypi vncserver-x11[492]: ConsoleDisplay: Found running X server (pid=753)
Oct  1 19:18:52 raspberrypi vncserver-x11[492]: AgentInitCheck: no response from agent
Oct  1 19:19:16 raspberrypi org.gtk.Private.AfcVolumeMonitor[1084]: Volume monitor alive
Oct  1 19:18:42 raspberrypi vncserver-x11[499]: ServerManager: Server started
Oct  1 19:18:44 raspberrypi vncserver-x11[499]: ConsoleDisplay: Cannot find a running X server on vt1
Oct  1 19:18:44 raspberrypi vncserver-x11[499]: AgentInitCheck: agent comms failure
Oct  1 19:18:46 raspberrypi vncserver-x11[499]: ConsoleDisplay: Found running X server (pid=709)
Oct  1 19:24:34 raspberrypi org.gtk.Private.AfcVolumeMonitor[895]: Volume monitor alive
Oct  1 19:18:42 raspberrypi vncserver-x11[496]: ServerManager: Server started
Oct  1 19:18:44 raspberrypi vncserver-x11[496]: ConsoleDisplay: Cannot find a running X server on vt1
Oct  1 19:18:44 raspberrypi vncserver-x11[496]: AgentInitCheck: agent comms failure
Oct  1 19:18:47 raspberrypi vncserver-x11[496]: ConsoleDisplay: Found running X server (pid=741)
Oct  1 19:18:52 raspberrypi vncserver-x11[496]: AgentInitCheck: no response from agent
Oct  1 19:49:18 raspberrypi org.gtk.Private.AfcVolumeMonitor[953]: Volume monitor alive
Oct  1 19:56:33 raspberrypi vncserver-x11[496]: Connections: connected: 10.255.255.249::52944
Oct  1 19:56:37 raspberrypi vncserver-x11[496]: Connections: authenticated: 10.255.255.249::52944, as pi (f permissions)
Oct  1 19:56:38 raspberrypi vncserver-x11[496]: SPrintConnMgr: Failed to add printer: server-error-service-unavailable
Oct  1 19:57:58 raspberrypi org.gtk.vfs.Daemon[953]: A connection to the bus can't be made
Oct  1 19:57:58 raspberrypi org.gtk.vfs.Daemon[953]: g_dbus_connection_real_closed: Remote peer vanished with error: Underlying GIOStream returned 0 bytes on an async read (g-io-error-quark, 0). Exiting.
Oct  1 19:58:15 raspberrypi iot[525]: **** IoT Raspberry Pi Sample has started ****
Oct  1 19:58:15 raspberrypi iot[525]: Config file not found. Going to Quickstart mode
Oct  1 19:58:15 raspberrypi iot[525]: Running in Quickstart mode
Oct  1 19:58:16 raspberrypi vncserver-x11[495]: ServerManager: Server started
Oct  1 19:58:18 raspberrypi vncserver-x11[495]: ConsoleDisplay: Cannot find a running X server on vt1
Oct  1 19:58:18 raspberrypi vncserver-x11[495]: AgentInitCheck: agent comms failure
Oct  1 19:58:20 raspberrypi vncserver-x11[495]: ConsoleDisplay: Found running X server (pid=749)
Oct  1 19:58:25 raspberrypi vncserver-x11[495]: AgentInitCheck: no response from agent
Oct  1 19:58:48 raspberrypi org.gtk.Private.AfcVolumeMonitor[1052]: Volume monitor alive
Oct  1 19:58:15 raspberrypi vncserver-x11[497]: ServerManager: Server started
Oct  1 19:58:17 raspberrypi vncserver-x11[497]: ConsoleDisplay: Cannot find a running X server on vt1
Oct  1 19:58:17 raspberrypi vncserver-x11[497]: AgentInitCheck: agent comms failure
Oct  1 19:58:19 raspberrypi vncserver-x11[497]: ConsoleDisplay: Found running X server (pid=707)
Oct  1 20:05:31 raspberrypi org.gtk.Private.AfcVolumeMonitor[894]: Volume monitor alive
Oct  1 19:58:15 raspberrypi vncserver-x11[496]: ServerManager: Server started
Oct  1 19:58:17 raspberrypi vncserver-x11[496]: ConsoleDisplay: Cannot find a running X server on vt1
Oct  1 19:58:17 raspberrypi vncserver-x11[496]: AgentInitCheck: agent comms failure
Oct  1 19:58:19 raspberrypi vncserver-x11[496]: ConsoleDisplay: Found running X server (pid=707)
Oct  1 19:58:39 raspberrypi org.gtk.Private.AfcVolumeMonitor[1050]: Volume monitor alive
